src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Proxy.cs

```diff
@@ -22,6 +22,31 @@ public abstract class HttpClientHandler_Proxy_Test : HttpClientHandlerTestBase
     {
         public HttpClientHandler_Proxy_Test(ITestOutputHelper output) : base(output) { }
 
+        [Fact]
+        public async Task Dispose_HandlerWithProxy_ProxyNotDisposed()
+        {
+            var proxy = new TrackDisposalProxy();
+
+            await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+            {
+                using (HttpClientHandler handler = CreateHttpClientHandler())
+                {
+                    handler.UseProxy = true;
```

Comment:
nit: .UseProxy defaults to `true`. But since this is test code, it's fine to leave as-is since it makes the intention clearer.